docs/releases/index.rst

```diff
@@ -6,7 +6,7 @@ Release notes
 
    upgrading
    5.3
-   5.2
+   5.2 LTS
```

Comment:
This should be left unchanged, as it needs to match the filename `docs/releases/5.2.md`.

Reply:
Oh, I didn't realize. I will correct this too.